seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/TimeFormat.java

```diff
@@ -29,4 +35,19 @@ public class TimeFormat {
                     .appendPattern("HH:mm:ss")
                     .appendFraction(ChronoField.NANO_OF_SECOND, 0, MAX_TIME_PRECISION, true)
                     .toFormatter();
+    public static final DateTimeFormatter CUSTOM_DATE_TIME_FORMAT =
```

Comment:
this format still is fixed, can't be customized.

Reply:
So far I've only added a common custom date format, the best way to handle this is to add parameters to JsonSchema, and all connectors that use JsonSchema will need to be tweaked and modified

Reply:
The file source/sink have `datetime_format` option, it can parse the user config's parameter, maybe you can use this config to do the customized.

https://github.com/apache/seatunnel/blob/dev/seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/BaseSinkConfig.java#L91

Reply:
I know that the text format has this configuration parameter, but the JSON format does not